### DISCUSSION TITLE
Changing the default edge type to 'smoothstep'

### DISCUSSION THREAD

**Discussion by Sameer4real** [xyflow/xyflow#2260] [Q&A] [answered]
Hey guys
I'm trying to create a custom edge with a button inside of it
I followed the documentation's instructions and it worked 
the thing that I want is to change the edge type to be 'smoothstep' 
Does anyone have an idea how can apply two types to the same edge?

or how to add type to a custom edge instead of the default edge type?

**Comment by bcakmakoglu** [comment, marked as answer]
You can't apply "two" edge-types at the same time.
If you use a custom edge and want the path to be a smoothstep path, use the `getSmoothStepPath` utility that is exported by react-flow.

[You can find more in the docs.](https://reactflow.dev/docs/api/edges/edge-utils/#getsmoothsteppath)

**Reply by Sameer4real** [reply]
Thank you man, I really appreciate it!
